src/components/Navbar.tsx

```diff
@@ -1,7 +1,13 @@
-import React from "react";
+import React, { useState } from "react";
 import { GymName } from "../constants";
 
 const Navbar = () => {
+  // set the default dropmenu as false because we want to hide it in smaller screen
+  const [dropMenu, setDropMenu] = useState(false);
+  // function for toggling drop menu
+  const toggleDropMenu = () => {
+    setDropMenu(!dropMenu);
```

Comment:
This should do the job, but this is not a good practice in the future when you are dealing with `useState` in React.

`useState` is an asynchronous hook and it doesn't change the state immediately, it has to wait for the component to re-render. If we have another places to dealing with `dropMenu`, we might suffer with race conditions.

Use prevState instead. 

```suggestion
    setDropMenu(prevState => !prevState);
```

If you want to learn more about this, please you can read this.
https://www.delftstack.com/howto/react/react-setstate-prevstate/

Reply:
- [ ] Change the code to use `prevState`

Reply:
Thank you for your input. 
- [x] Change the code to use ```prevState```